nservicebus/gateway/index.md

```diff
@@ -69,13 +66,46 @@ A gateway runs inside each host process. The gateway gets its input from a regul
 
 ### Configuring the gateway
 
-In Version 5 the gateway is provided by the `NServiceBus.Gateway` NuGet. In Version 3 and Version 4 the gateway is included in the core assembly, meaning that every endpoint is capable of running a gateway.
+In Version 5 and above the gateway is provided by the `NServiceBus.Gateway` NuGet. In Version 3 and Version 4 the gateway is included in the core assembly, meaning that every endpoint is capable of running a gateway.
```

Comment:
"meaning that every endpoint is capable of running a gateway" this is true in all versions. so redundant and should be removed